app/src/main/java/at/bitfire/icsdroid/model/SubscriptionsModel.kt

```diff
@@ -60,10 +63,28 @@ class SubscriptionsModel(application: Application): AndroidViewModel(application
         .stateIn(viewModelScope, SharingStarted.WhileSubscribed(5000), AppAccount.DEFAULT_SYNC_INTERVAL)
 
     init {
+        // Migrate existing SharedPreferences to DataStore
+        migrateSharedPreferencesToDataStore()
         // When initialized, update the ask* fields
         checkSyncSettings()
     }
 
+    private fun migrateSharedPreferencesToDataStore() = viewModelScope.launch(Dispatchers.IO) {
```

Comment:
It's only for the dark theme setting no? I think we can drop it. It's not pretty to have it here and people can simply re-enable it without much effort. Like you said, it's not worth it really.